lib/guardian/utils.ex

```diff
@@ -3,7 +3,7 @@ defmodule Guardian.Utils do
   @doc false
   def stringify_keys(nil), do: %{}
   def stringify_keys(map) do
-    Enum.reduce(Dict.keys(map), %{}, fn(k,acc) -> Dict.put(acc, to_string(k), map[k]) end)
+    Enum.reduce(Map.keys(map), %{}, fn(k,acc) -> Map.put(acc, to_string(k), map[k]) end)
```

Comment:
This first one may not be a map. The one inside the anon function is for sure, but the other... not so sure.